packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -53,59 +56,70 @@ function NotificationItem({
   const hasAvatar = avatarComponents.length > 0;
 
   return (
-    <div
-      className={classNames(
-        'relative group flex flex-row py-4 pl-6 pr-4 hover:bg-theme-hover focus:bg-theme-active border-y border-theme-bg-primary',
-        isUnread && 'bg-theme-float',
+    <ConditionalWrapper
+      condition={!!onClick}
+      wrapper={(children) => (
+        <Link href={targetUrl}>
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+          <a
+            data-testid="openNotification"
+            role="button"
+            title="Open notification"
+            aria-label="Open notification"
+            onClick={onClick}
+            tabIndex={0}
+          >
+            {children}
+          </a>
+        </Link>
       )}
     >
-      {onClick && (
-        <button
-          type="button"
-          aria-label="Open notification"
-          className="absolute inset-0 z-0"
-          onClick={onClick}
-        />
-      )}
-      {onOptionsClick && (
-        <OptionsButton
-          className="hidden group-hover:flex top-3 right-2"
-          position="absolute"
-          type="button"
-          onClick={onOptionsClick}
-        />
-      )}
-      <NotificationItemIcon
-        icon={icon}
-        iconTheme={notificationTypeTheme[type]}
-      />
-      <div className="flex flex-col flex-1 ml-4 w-full text-left typo-callout">
-        {hasAvatar && (
-          <span className="flex flex-row gap-2 mb-4">{avatarComponents}</span>
+      <div
```

Comment:
I agree that we should turn this into a link/anchor tag, though the reason we made the recent change to div, there was another anchor tag nested here. What we need to do here is just change the `button` element to an anchor tag.

Also, every item in the notifications table, there would be an `onClick` value passed down for tracking purposes.

Reply:
I think Lee is right, we should leverage the existing button to be converted to a `a tag` to maintain the order of having multiple a tags spanned here.

Reply:
Yeah makes sense, was checking the implementation in history to make it consistent, but probably the issue you mentioned Lee on having nested a tags was not there. 
Cool, I'll just convert from using the button there to using an `a tag`. 

@sshanzel I'm not sure I understood correctly what you ment by:

> Also, for every item on the notifications page, there would be an onClick value passed down for tracking purposes. Meaning, they would always turn out to be not a link.

I've kept the onClick functionality there, so clicking the `a tag` would fire onClick as well in order to send the analytics event. Did you mean something else I've missed?

Reply:
Oh, I just went back and checked the condition, thought it was the other way around about making it an anchor tag or a button. Kindly disregard that part 🚢